src/components/Card/Card.tsx

```diff
@@ -114,6 +127,39 @@ export function CardActions(props: React.PropsWithChildren<CardActionsProps>) {
   return children && <Box __css={styles}>{children}</Box>;
 }
 
+/**
+ * If `mainActionLink` is passed, then this adds Chakra's `LinkBox` wrapper
+ * component and `LinkOverlay` component to allow the entire `Card` component
+ * to be clickable. Otherwise, the initial `Card` component is returned. Make
+ * sure to pass text in the `mainAriaLabel` to make the link accessible.
+ */
+export function CardLinkBox(props: React.PropsWithChildren<CardLinkBoxProps>) {
+  const { children, mainActionLink, mainAriaLabel } = props;
+  // This allows images in the Card to be clickable.
+  const zIndex = { zIndex: "9999" };
+
+  // Since we are adding a link with no discernable text, we must
+  // add an aria-label to keep the `Card` component accessible.
+  if (!mainAriaLabel) {
+    console.warn(
+      "Pass in text for the `mainActionLink` URL in the `mainAriaLabel` prop for accessibility."
+    );
+  }
+
+  return mainActionLink ? (
+    <ChakraLinkBox>
+      <ChakraLinkOverlay
+        href={mainActionLink}
+        _before={zIndex}
+        aria-label={mainAriaLabel}
+      />
+      {children}
```

Comment:
I believe wrapping everything in the `Card` in the `LinkOverlay` does not allow the internal buttons and links to be mouse clickable unless you tab to them.  What do you think about only wrapping `cardContent` in `ChakraLinkOverlay`?  That would align with the example on the Chakra site where the only heading is wrapped and the buttons are outside.

Reply:
Yeah, I'll try that. It makes more sense.